x509/src/certificate.rs

```diff
@@ -25,6 +28,57 @@ pub fn default_zero() -> u32 {
 /// Version  ::=  INTEGER  {  v1(0), v2(1), v3(2)  }
 pub const X509_CERT_VERSION: u8 = 2;
```

Comment:
This seems like it could either just be `(X509)Version::default()` or an inherent constant

Reply:
I meant to remove that when i added the enum. Do you think we need to retain it as a default? if not, i will just delete.

Reply:
I think adding a `Default` impl would be a good idea

Reply:
Added